### DISCUSSION TITLE
Subtitle on Top and Bottom

### DISCUSSION THREAD

**Discussion by lordCuppa** [SubtitleEdit/subtitleedit#6719] [Q&A]
Hi,
I just started to play around with SubtitleEdit and I have to say I really like it. But one of the core function I missed during my first finished translation of an episode, is to have a text (subtitle) on Top and Bottom in the same timeline. So I would like to know if I just haven't found this function yet, or there isn't one. And if so, is there a workaround to achieve it?
Thanks in advance.
![Example](https://user-images.githubusercontent.com/98316739/222986258-5d5b6b79-4d1f-4127-87ac-429bdb0dede9.png)

**Comment by darnn** [comment]
SE supports the tags in SRT files that can be used to place subtitles on various parts of the screen, see here for instance:
https://superuser.com/questions/642265/moving-subtitle-text-up-in-srt-file-format
However, you can't change the tag in the middle of the subtitle. So if you really wanted to do this in an SRT file, your best bet would be to have two subtitles with the same timecode, one for the top line and one for the bottom.
Conversely you can use SSA/ASS, but I don't know anything about the formatting there, I would just assume that it would be easier to achieve. But maybe it is just the same method, I have no idea.

Your other option, of course, is to simply use a ton of linebreaks between the two lines, so that one ends up on top and one on the bottom. Not the most elegant, but gets the job done.

**Reply by lordCuppa** [reply]
@darnn For my first subtitle I tried out the ASS format and didn't find a solution that worked. Now I would try SRT, and your tip with overlapping the subtitles, which looks like it works. The only thing that happens, is that SubtitleEdit don't like it, and you need to watch out to not interact too much with it in the waveform, or it will jump apart from each other. Do you probably know a shortcut to quickly get the time code from the above one?
Example Solution:
![grafik](https://user-images.githubusercontent.com/98316739/222987735-9cab8f27-dd2e-42d5-9dcd-4bd462c90044.png)
Example Shortcut-Question:
![Screenshot 2023-03-05 224107](https://user-images.githubusercontent.com/98316739/222987629-6afdc60b-4cd5-4ea8-b29c-adfd7e98c622.png)

**Reply by darnn** [reply]
If it were me, I would just copy the first subtitle and paste it (to get one with the same timecode) and then paste whatever text I wanted into it. Note that this can only be done properly when you're editing a single subtitle, rather than an original and a translation, because copying and pasting whole subtitles there causes a desync.

If you want specifically to copy and paste the timecode, copy the subtitle you want, then select the subtitle you want to paste into in list view, and use "column(,) paste". You need to configure a keyboard shortcut for it in options->settings->shortcuts, it's under list view and it's called "column, paste". When you use this keyboard shortcut, select "time codes only"  and "overwrite".
